### ISSUE TITLE
[Faults] - Make faults bitwise and stackable

### ISSUE THREAD

**Issue by jr1221** [Northeastern-Electric-Racing/Cerberus#226]
### Description

This would revamp faults a little bit.  Basically faults in shep are bitwise, so each bit is a fault.  This allows faults to be stacked such that multiple faults could be broadcasted.

### Acceptance Criteria

High priority faults trigger faulting as normal and state change.
Last high priority fault--> 4 secs --> unfault
Low priority fault --> 4 secs --> clear it.

Big trouble here is how to clear low priority faults without a shit load of timers.  Ew.

### Proposed Solution

Make it so faults are bitwised onto each other. Make the severity and fault level of MPU be based off of the fault with the highest severity. Make logic such that unfualt occurs only after 4 sec of most recent highest severity fault (reset timer on all high faults).  Clear low severity faults on a timer irregardless of high priority faults, but implement similar logic
